src/analytics/components/AnalyticsFilter.react.js

```diff
@@ -0,0 +1,61 @@
+import {Grid, IconButton, InputLabel} from '@mui/material';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import FormControl from '@mui/material/FormControl';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import * as React from 'react';
+
+export default function AnalyticsFilter() {
+  return (
+    <Box sx={{my: '24px'}}>
+      <Grid container spacing={1}>
+        <Grid item xs={4.5}>
+          <FormControl fullWidth>
+            <InputLabel id="phases-select">Phases</InputLabel>
+            <Select label="Phases">
+              <MenuItem value="">
+                <em>None</em>
+              </MenuItem>
+              <MenuItem value={'Translation'}>Translation</MenuItem>
+              <MenuItem value={'Grammer'}>Grammar</MenuItem>
```

Comment:
```suggestion
              <MenuItem value="Translation">Translation</MenuItem>
              <MenuItem value="Grammer">Grammar</MenuItem>
```

Use cases like this can be HTML strings instead. There are other cases in this file, make sure you don't forget about them tho.